packages/broadcast/component.js

```diff
@@ -22,6 +26,7 @@ export class FabricBroadcast extends LitElement {
     super();
     this._messages = [];
     this.interval = 30000;
+    this.hiddenMessageIds = [];
```

Comment:
not a big deal but idiomatic Lit seems to be to use an underscore in the name for private state variables such as this one so it would be something like this._hiddenMessageIds

Reply:
I see we also use underscore on `_message`, so definitely agree with this as we should aim to be as consistent as possible with this kinda stuff 😁

Reply:
Fixed in [9da5ddb](https://github.com/fabric-ds/elements/pull/61/commits/9da5ddbf339fa80aa2226fa0ec4834263b006ee3)